src/components/ChangeTodosStatusButton/ChangeTodosStatusButton.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import React, { useMemo } from 'react';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  loadTodos: () => Promise<void>;
+  onUpdateTodoStatus: (todoId: number, status: boolean) => Promise<void>;
+  onChangeProcessingIds: (todoId: number | []) => void;
```

Comment:
number or array, that's interesting 🙂

Reply:
I created a method that can clear the processingIds or add a new id to the array of ids. So, I don't have other ideas:(